scripts/test-truncating-of-client-hello.py

```diff
@@ -136,66 +133,86 @@ def main():
                                 ("hello truncate", -13, 0),
                                 ]:
 
-        conversation = Connect(host, port)
-        node = conversation
+        conversation_name = None
+        if "pad" in name:
+            conversation_name = name + ": " + str(pad_len) + " of \"" + str(pad_byte) + "\" byte padding"
```

Comment:
if we're doing clean up, we should use the recommended [`format`](https://docs.python.org/2/library/stdtypes.html#str.format), not string concatenation